sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -189,6 +190,8 @@ protected String checkIfOverwritten(String key, String param) {
                         case TYPE_DISTRO:
 	                    case TYPE_CONFIG:
                             return properties.getProperty(PROPERTY_DISTRO_GROUP_ID, Artifact.GROUP_DISTRO);
+                        case TYPE_CONTENT:
+                            return properties.getProperty(param + ".groupId",Artifact.GROUP_CONTENT);
```

Comment:
Can we get this to a closer parallel to `TYPE_OMOD`? Here we should only need to handle the default value.